packages/three-vrm-animation/package.json

```diff
@@ -36,7 +36,7 @@
   "scripts": {
     "version": "yarn all",
     "all": "yarn lint && yarn clean && yarn build && yarn docs",
-    "dev": "cross-env NODE_ENV=development rollup -w -c",
+    "dev": "cross-env NODE_ENV=development SERVE=1 node ../../bin/build.mjs",
```

Comment:
dev command can replaced with `SERVE=1 yarn build-dev`?

If this is stil necessary I'd perfer 
`cross-env SERVE=1 yarn build-dev` like `build-dev` to reuse command definition since `yarn build-js` can be move to workspace root once package manager support.

Reply:
addressed @ https://github.com/pixiv/three-vrm/pull/1413/commits/46b2b2d53b68e380a0fabd2ad386aae283fffa7f